demos/1_Introduction/exercises/03_where_is_my_reader.ipynb

```diff
@@ -285,7 +285,10 @@
     "\n",
     "You will have to set the position of the source, rotation axis (object) and detector. Look at the [documentation of `create_Cone2D`](https://tomographicimaging.github.io/CIL/nightly/framework/#cone2d-geometry-fanbeam) as well as the notebook [00_CIL_geometry](../00_CIL_geometry.ipynb) to help you.\n",
     "\n",
-    "1. Create a fan-beam geometry using the `AcquisitionGeometry.create_Cone2D`\n",
+    "1. Create a fan-beam geometry using the `AcquisitionGeometry.create_Cone2D`. Note that in CIL, we define the **source to detector distance along the y axis** - be mindful of this when setting up your geometry. E.g. see this illustration of a 3D cone beam geometry, with the axes shown:\n",
```

Comment:
I would maybe say ` we define the **source to object and source to detector distance along the y axis**`

Reply:
has been updated now :)